gammapy/estimators/parameter.py

```diff
@@ -211,10 +221,17 @@ def estimate_scan(self, datasets, parameter):
             parameter=parameter,
             reoptimize=self.reoptimize
         )
+        success = True
+        for res in profile["fit_results"]:
+            success &= res
```

Comment:
This seems equivalent to `success = False not in profile["fit_results"]`
and redondant with : https://github.com/gammapy/gammapy/blob/832eb734cb6ac7c3d9902bfa454e323fc445a8e6/gammapy/estimators/parameter.py#L366

Reply:
Not exactly.... in parameter.py, one bin can fail. 
then, in the 'run' function I concatenate the fit_status for all estimations (e.g. scan, ul, errn-errp)